### DISCUSSION TITLE
Can you help me with undestanding the ouput?

### DISCUSSION THREAD

**Discussion by emindeniz** [jdepoix/youtube-transcript-api#258] [Q&A]
After running the following code:
```
from youtube_transcript_api import YouTubeTranscriptApi

transcript = YouTubeTranscriptApi.get_transcript('CizwH_T7pjg')

```
I am getting the following output:

[{'text': 'why did it all turn out like this for me',
  'start': 0.32,
  'duration': 5.439},
 {'text': 'i had so much promise', 'start': 2.879, 'duration': 5.441},
 {'text': 'i was personable i was bright', 'start': 5.759, 'duration': 3.361},
 {'text': 'oh', 'start': 8.32, 'duration': 4.08} ....

What does the duration mean in the output? For the first one it can't be 5.4 seconds because it is too long and conflicts with the next one. It can't be 5.4 milliseconds either because it is too short.  Watching the video and also based on the output, the first duration should be like 2 seconds. I'll appreciate if anybody can help me that.

**Comment by emindeniz** [comment]
This is the answer:

https://github.com/jdepoix/youtube-transcript-api/issues/21

Any idea how to get around this to get the exact duration?

**Reply by jdepoix** [reply]
Hi @emindeniz, there's no way around this. YouTube doesn't provide this information. The best you can do is taking the minimum of start + duration and start of the following snippet, which only will be a rough estimate.
